app/Http/Controllers/AsistenController.php

```diff
@@ -62,6 +63,26 @@ public function store(Request $request)
         return '{"message": "success"}';
     }
 
+    public function updateDesc(Request $request){
+        $user = Auth::guard('asisten')->user();
+        $request->validate([
+            'deskripsi' => 'required|min:6|max:150',
+        ]);
+
+        $asisten = Asisten::where('id',$user->id)->update([
+            'deskripsi'=>"$request->deskripsi",
+        ]);
+
+        // echo $asisten;
+        return response()->json([
+            'message'=> 'success',
+            'asisten' => $asisten,
+            'deskripsi' => $request->deskripsi,
+        ], 200);
+
+        return route('/');
```

Comment:
@fakhrip I forget is this safe to do? Just simply go to the `/` ?

Reply:
i forget to delete that route, i think this is safe but in flow, that syntax is useless because i've already create inertia in asisten.vue 😅

Reply:
p.s it is safe and okay

And i think this problem should be resolved by now, im resolving this conversation and feel free to reopen it again in case of any problem regarding this one